### ISSUE TITLE
How to use 68 landmarks in cvlib

### ISSUE THREAD

**Issue by ThanhNhann** [arunponnusamy/cvlib#19]
In the cvlib package, is there a method to use file `shape_predictor_68_face_landmarks.dat`?

**Comment by 7thstorm** [comment]
will this return the actual number of landmarks detected? I'm using dlib now and it is returning all 68 points even for profile faces. It should only return the actual number of landmarks detected. comment?

**Comment by ThanhNhann** [comment]
Yeah I want to have the coordinates of the points in faces but when i read the document of cvlib, It doesn't has.

**Comment by arunponnusamy** [comment]
@ThanhNhann as of now cvlib doesn't have method for landmark detection. You can use dlib directly to get the facial landmarks.

**Comment by arunponnusamy** [comment]
@7thstorm dlib landmark detector is actually trained on frontal faces I believe. It will try to give all 68 points for each face. There is also 5 point landmark detector in dlib. Maybe you can give that a try.
https://github.com/davisking/dlib-models
https://www.pyimagesearch.com/2018/04/02/faster-facial-landmark-detector-with-dlib/
